src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,130 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-xxs);
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-3x);
+  --sbb-card-padding-block-end: var(--sbb-card-padding-block-start);
+
+  position: relative;
+  overflow: hidden;
+  border-radius: var(--sbb-border-radius-4x);
+  transition: box-shadow var(--sbb-animation-duration-6x) var(--sbb-animation-easing);
+  z-index: 0;
+
+  &::before,
+  &::after {
+    position: absolute;
+    content: '';
+    inset: 0;
+  }
+
+  &::before {
+    background-color: var(--sbb-color-white-default);
+    z-index: -3;
+  }
+
+  &::after {
+    background-color: transparent;
+    z-index: -1;
+  }
```

Comment:
as discussed, try to move the definitions to an inner div or to the a / button tag if possible